rom/dev/tests/test_idevid_derivation.rs

```diff
@@ -55,7 +58,7 @@ fn test_idev_subj_key_id_algo() {
         hw.upload_firmware(&image_bundle.to_bytes().unwrap())
             .unwrap();
 
-        let result = hw.copy_output_until_exit_success(&mut output);
-        assert!(result.is_ok());
+        // Step until runtime firmware denotes successful firmware upload
+        hw.step_until(|m| m.soc_mbox().status().read().status().cmd_complete());
```

Comment:
The ROM still has plenty of stuff to do after the mailbox transaction is completed. We should probably be waiting until the runtime firmware has printed something out to the UART, or perhaps just send the runtime firmware a mailbox command to make sure it is alive.

Reply:
Done.